docs/HACKING.md

```diff
@@ -1,104 +1,181 @@
+
+```shell script
+sed -i 's/OpenShiftSDN/OVNKubernetes/g' openshift-data/install-config.yaml
+```
+
+Once done, your `install-config.yaml` should look like
+
+```shell script
+networking:
+  clusterNetwork:
+      - cidr: 10.128.0.0/14
+        hostPrefix: 23
+      machineNetwork:
+      - cidr: 10.0.0.0/16
+      networkType: OVNKubernetes
+```
+
+#### Generate Manifests
+
+The manifests also must be changed in order to enable OVNKubernetes networking.
+
+Follow the docs
+[here](https://docs.openshift.com/container-platform/4.12/networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.html)
+for configuration steps.
+
+#### Launch the cluster
+
+Once the prerequisite steps are completed, launch your cluster for testing.
 
 ## Build
-To build and push the operator image, execute:
+
+Now that your cluster is running, the WMCO can be built and deployed. To manually build, see the subheading
+[below](#build-an-operator-image). To build and deploy automatically using the OLM or the e2e hack script, jump ahead to
+[Deploy](#deploy)
+
+### Build an operator image
+
+If you want to deploy the operator manually without the OLM, you need to build an operator image that we can deploy to
+the cluster.
+From the top level of your wmco directory, run
+
 ```shell script
-export OPERATOR_IMAGE=quay.io/<insert username>/wmco:$VERSION_TAG
-hack/olm.sh build
+podman build -t quay.io/<quay username>/wmco:<operator name> -f build/Dockerfile .
 ```
 
 Building the full WMCO image is a time consuming process given we are building all the submodules. If you are just
 changing WMCO and not the submodules, you can separately build the submodules as a base image and then just build the
 WMCO  image from this base image:
+
 ```shell script
 make base-img
 make wmco-img IMG=$OPERATOR_IMAGE
 ```
+
 `make base-img` will not push the base image as a push is not needed. `make wmco-img` will make and push the
 `$OPERATOR_IMAGE`
 
-## Development workflow
-With the Operator image built and pushed to a remote repository, the operator can be deployed either directly,
-or through OLM.
+### Push an operator image
+
+Push your image to your container registry with podman
+
+`podman push quay.io/<quay username>/wmco:<operator name>`
+
+## Deploy
+
+Before deploying, ensure your environment is properly configured.
+
+#### Set private key
+
+in order to SSH into the instance, you'll need to set the `KUBE_SSH_KEY_PATH` to your private key.
 
-For each deployment method a kubeconfig for an OpenShift or OKD cluster must be exported
 ```shell script
-export KUBECONFIG=<path to kubeconfig>
+export KUBE_SSH_KEY_PATH=<path to pem>
 ```
 
-### Deploying the operator directly
+#### Set Image Name
+
+This is the name of the operator image that will be pushed to the container registry.
+
 ```shell script
-make deploy IMG=$OPERATOR_IMAGE
+export OPERATOR_IMAGE=quay.io/<quay username>/wmco:<identifying name>
```

Comment:
add underscore
'identifying_name'

Reply:
Maybe it should be identifying_tag instead?

Reply:
tbh I've always found adding underscores instead of spaces to be a little more confusing because to me it feels like it should be a variable of some sort when it's not :thinking: